docker/api/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu:latest
+RUN apt-get update -y
+RUN apt-get install -y python-pip python-dev build-essential
```

Comment:
Is this actually required? There might be python images that you could leverage. Also not a show stopper.

Reply:
I believe there is a base python container. I will look into moving over to that one. My goal is to get the basics up first, then improve from there.